src/runtime/core/CMakeLists.txt

```diff
@@ -160,6 +160,6 @@ INSTALL(FILES
       ${CMAKE_CURRENT_SOURCE_DIR}/../../../bin/mygpu
       ${CMAKE_CURRENT_SOURCE_DIR}/../../../bin/gputable.txt
       PERMISSIONS OWNER_EXECUTE OWNER_READ GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
-      DESTINATION "bin"
+     DESTINATION "libexec/atmi"
```

Comment:
Given that these binaries conflict with aomp-extras and/or hipfort, do you just want to comment out these lines (do not remove, just comment out)? Do you want to do it as part of the same PR so it is easy to track all changes in one place?

Reply:
Sure , will comment out the installation of binaries